Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.10-alpine
+LABEL maintainer="leon.kushnir15@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
+RUN mkdir -p /files/media
+
+RUN adduser \
+    --disabled-password \
+    --no-create-home \
+    my_user
+
+RUN chown -R my_user /files/media
+RUN chmod -R 755 /files/media

```

Comment:
It's generally a good practice to set permissions and ownership in a single RUN command to reduce the number of layers in the Docker image. Consider combining these two RUN commands.